content/ja/handbook/engineering/recognition.md

```
@@ -0,0 +1,267 @@
+upstream_path: /handbook/engineering/recognition/
+upstream_sha: 0e6f01390a34aeb6706ace17d8d3c50e74e82d0d
+translated_at: "2026-04-29T00:00:00Z"
+translator: claude
+stale: false
+---
+
+## エンジニアリング四半期ごとの優秀者表彰
+
+四半期ごとに、CTO リーダーシップはその四半期で卓越した活躍をしたエンジニアリングチームメンバーを表彰します。受賞者はピアからの推薦から選ばれます。受賞者はエンジニアリング全社会議で発表されます。表彰には以下が含まれます:
+
+- [エンジニアリング四半期優秀者チャット](#engineering-quarterly-achievers-chat)への招待
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)